chainer/optimizer_hooks/gradient_lars.py

```diff
@@ -0,0 +1,82 @@
+from chainer import cuda
+
+
+class GradientLARS(object):
+
+    """Optimizer/UpdateRule hook function for layer wise adaptive rate scaling.
+
+    See: `Large Batch Training of Convolutional Networks \
+          <https://arxiv.org/abs/1708.03888>`_
+       : `Convergence Analysis of Gradient Descent Algorithms \
+          with Proportional Updates \
+          <https://arxiv.org/abs/1801.03137>`_
+
+    This hook function scales all gradient arrays to fit to the weight norm.
+
+    In <https://arxiv.org/abs/1708.03888>,
+        v_{t+1} = m * v_t + \\gamma * \\lambda * (\\nabla L(w_t) + \\beta w_t)
```

Comment:
Could you use `.. math::` directive ([document](http://www.sphinx-doc.org/en/master/ext/math.html#directive-math), [example](https://github.com/chainer/chainer/blob/3ba7d4d0ddcc0202225161f75e6d6fed07ffa25e/chainer/functions/noise/gumbel_softmax.py))?

Reply:
You can check if the directive works by compiling the document. We can compile the document by typing 
`sphinx-build -b html -d build/doctrees source build/html` at `docs` directory in the Chainer repository.